src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -0,0 +1,266 @@
+  }, [addSubscription, context, context.target, context.api, setIsLoading, handleTemplates, handleError]);
+
+  React.useEffect(() => {
+    addSubscription(
+      context.target.target().subscribe(() => {
+        setFilterText('');
+        refreshTemplates();
+      })
+    );
+  }, [context, context.target, addSubscription, refreshTemplates]);
+
+  React.useEffect(() => {
+    if (!context.settings.autoRefreshEnabled()) {
+      return;
+    }
+    const id = window.setInterval(
+      () => refreshTemplates(),
+      context.settings.autoRefreshPeriod() * context.settings.autoRefreshUnits()
+    );
+    return () => window.clearInterval(id);
+  }, []);
+
+  React.useEffect(() => {
+    const sub = context.target.authFailure().subscribe(() => {
+      setErrorMessage('Auth failure');
+    });
+    return () => sub.unsubscribe();
+  }, [context.target]);
+
+  const displayTemplates = React.useMemo(
+    () => templates.map((t: EventProbe) => [t.id, t.name, t.clazz, t.description, t.methodName + t.methodDescriptor]),
+    [templates]
+  );
+
+  const handleModalToggle = () => {
+    addSubscription(
+      context.api
+        .removeProbes()
+        .pipe(first())
+        .subscribe(() => {
+          refreshTemplates();
+        })
+    );
+  };
+
+  React.useEffect(() => {
+    addSubscription(
+      context.notificationChannel
+        .messages(NotificationCategory.ProbeTemplateApplied)
+        .subscribe((v) => refreshTemplates())
+    );
+  }, [addSubscription, context, context.notificationChannel, setTemplates]);
+
+  const authRetry = React.useCallback(() => {
+    context.target.setAuthRetry();
+  }, [context.target, context.target.setAuthRetry]);
+
+  if (errorMessage != '') {
+    return (
+      <ErrorView
+        title={'Error retrieving events'}
+        message={
+          'This is commonly caused by the agent not being loaded/active, check that the target was started with the agent (-javaagent:/path/to/agent.jar) ' +
+          errorMessage
+        }
+        retry={isAuthFail(errorMessage) ? authRetry : undefined}
+      />
+    );
+  } else if (isLoading) {
+    return <LoadingView />;
+  } else {
+    return (
+      <>
+        <Card>
+          <CardHeader>
+            <CardHeaderMain>
+              <Text component={TextVariants.h4}>About the JMC Agent</Text>
+            </CardHeaderMain>
+          </CardHeader>
+          <CardBody>
+            The JMC Agent allows users to dynamically inject custom JFR events into running JVMs. In order to make use
+            of the JMC Agent, the agent jar must be present in the same container as the target, and the target must be
+            started with the agent (-javaagent:/path/to/agent.jar). Once these pre-requisites are met the user can
+            upload probe templates to Cryostat and insert/remove them from targets, as well as view currently active
+            probes.
+          </CardBody>
+        </Card>
+        <Toolbar id="event-templates-toolbar">
```

Comment:
should have a different id? `event-templates-toolbar` is already in use.

https://github.com/cryostatio/cryostat-web/blob/e41e2c8cb1d8bacb004721e10b2d8e5cf84603e7/src/app/Events/EventTemplates.tsx#L365